srcpkgs/python3-publicsuffix2/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-publicsuffix2'
+pkgname=python3-publicsuffix2
+version=2019.12.21
+revision=1
+wrksrc="python-publicsuffix2-release-2.${version//./-}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip python3-wheel"
```

Comment:
why pip?

Reply:
https://github.com/void-linux/void-packages/runs/7171655428?check_suite_focus=true#step:7:1299

Reply:
```
Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmp3mvkx0wo', '--quiet', 'requests>=2.7.0']' returned non-zero exit status 1.
```
That only means you are missing requests from dependencies. Adding pip to hostmakedepends will just hide this issue and download requests in their preferred version, but we want to run the tests on Void packages.

Reply:
![2022 07 04-09:05:15](https://user-images.githubusercontent.com/22344340/177058767-fec7e338-cf4f-4140-a66f-3180cbc73483.png)